### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bitbrain-za**
Thank you for the assistance.

I've also added the steps I took to get everything running.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ This layer depends on:
 
 Follow the usual steps to setup OpenEmbedded and bitbake.
 
+Steps:
```
Comment (alistair23) [inline README.md:28]: Can you change this to "One option to simplify setup is to use poky, these are some example steps to do this:"? Not everyone (including me) uses poky.

File: README.md [hunk 2]
```diff
@@ -93,3 +101,13 @@ DISTRO_FEATURES += "x11"
 ## Demos
 
 Below are some demos of this layer being used with Pine64 boards.
+
+## Deploy to SD card
+
+First check your SD card fd using `lsblk`.
```
Comment (alistair23) [inline README.md:107]: file path instead of fd?

File: README.md [hunk 3]
```diff
@@ -93,3 +101,13 @@ DISTRO_FEATURES += "x11"
...
+```
+cd tmp/deploy/images/pine-a64-lts/
```
Comment (alistair23) [inline README.md:109]: Can you point out that this is for Linux?

File: README.md [hunk 4]
```diff
@@ -25,6 +25,14 @@ This layer depends on:
 
 Follow the usual steps to setup OpenEmbedded and bitbake.
 
+Steps: 
+```
```
Comment (alistair23) [inline README.md:29]: Can you use ```shell

**Review by alistair23 — CHANGES_REQUESTED** [review]
Thanks! I have a few comments

**Comment by bitbrain-za** [comment]
Thank you for the comments. Will make the changes now.

**Review by alistair23 — APPROVED** [review]

**Comment by alistair23** [comment]
Looks good, do you mind squashing the two commits into 1. You can then force push to the branch

**Comment by bitbrain-za** [comment]
> Looks good, do you mind squashing the two commits into 1. You can then force push to the branch

I think that's done on your side. There should be a squash option with the merge?

**Comment by alistair23** [comment]
I would prefer to just rebase and merge a single commit.

**Comment by bitbrain-za** [comment]
Rebase and merged commits

**Comment by alistair23** [comment]
Thanks. In future you can just force push to the current branch.

I know it may seem strange if you only use the GitHub interface, but by addressing comments in the original commit it makes a projects history much cleaner and maintains bisectability.

**Pull request closed without merging**